index.js

```diff
@@ -5,7 +5,7 @@ const unusedFilename = require('unused-filename');
 
 const app = electron.app;
 
-function registerListener(win, opts = {}) {
+function registerListener(win, opts = {}, cb) {
```

Comment:
What do you think should happen to the Promise in the `! unregisterWhenDone` code path? With a Promise we'd have to explicitly guard against resolving more than once, right?